src/sdk/namespace_fs.js

```diff
@@ -890,7 +892,8 @@ class NamespaceFS {
                 const entry_path = path.join(this.bucket_path, r.key);
                 //If entry is outside of bucket, returns stat of symbolic link
                 const use_lstat = !(await this._is_path_in_bucket_boundaries(fs_context, entry_path));
-                r.stat = await nb_native().fs.stat(fs_context, entry_path, { use_lstat });
+                const result_stat = await native_fs_utils.stat_ignore_enoent(fs_context, entry_path, { use_lstat });
```

Comment:
why ignore just enoent? if we got a different error we will fail the entire list op again

Reply:
Do you want to on the second `stat` to ignore all errors? I can do that